Tensile/Common.py

```diff
@@ -69,6 +69,29 @@
 globalParameters["SyncsPerBenchmark"] = 1         # how iterations of the stream synchronization for-loop to do per benchmark data point
 globalParameters["EnqueuesPerSync"] = 1           # how many solution enqueues to perform per synchronization
 globalParameters["SleepPercent"] = 300            # how long to sleep after every data point: 25 means 25% of solution time. Sleeping lets gpu cool down more.
+globalParameters["FlushCount"] = 1                # Number of copies of arrays to allocate for cache flushing in timing code.
+                                                  # Functions are called iters times in a timing loop.
+                                                  # If the problem memory footprint is small enough, then arrays will be cached.
+                                                  # flush_count can be used to prevent caching.
+                                                  # For example, for sgemm with transA=transB=N:
+                                                  # problem_memory_footprint = (m*k + k*n + m*n) * sizeof(float).
```

Comment:
Maybe using rotate buffer for D (write only) is not necessary?
Is this same as rocblas?

Reply:
Yes we considered this and decided since it is write only it may not be necessary. We are not 100% sure of any speed up for write only when using MALL. In any case this is the more conservative method to ensure that we are creating more buffers rather than less. We can revisit at a later point if there is more clarity, same as rocBLAS for now